renovate-config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Default configuration for repositories in the Laminas organisation",
+    "extends": [
+        ":enableRenovate",
+        ":dependencyDashboard",
+        ":ignoreModulesAndTests",
+        ":automergeMinor",
+        ":automergeBranch",
+        ":prNotPending",
+        ":rebaseStalePrs",
+        ":separateMajorReleases",
+        ":combinePatchMinorReleases",
+        ":enableVulnerabilityAlerts",
+        ":timezone(UTC)",
+        ":label(renovate)",
+        "group:allNonMajor",
+        "schedule:daily"
+    ],
+    "enabledManagers": ["composer", "dockerfile", "docker-compose", "github-actions"],
+    "lockFileMaintenance": {"enabled": true, "extends": ["schedule:daily"]},
+    "platformAutomerge": true,
+    "prFooter": "[Read more information](https://github.com/laminas/.github/blob/main/RENOVATE.md) about the use of [Renovate Bot](https://github.com/renovatebot/renovate) within Laminas.",
+    "rangeStrategy": "replace",
+    "rollbackPrs": true,
+    "packageRules": [
+        {"matchDepTypes": ["require"], "rangeStrategy": "widen"},
+        {"matchPackagePatterns": ["^laminas/"], "groupSlug": "laminas", "groupName": "Laminas packages"},
+        {"matchPackagePatterns": ["^laminas-api-tools/"], "groupSlug": "laminas-api-tools", "groupName": "Laminas API Tools packages"},
+        {"matchPackagePatterns": ["^mezzio/"], "groupSlug": "mezzio", "groupName": "Mezzio packages"}
```

Comment:
You can use a `description` field to explain what each `packageRules` object does. 😉

Example from one of my own Renovate configs:

```json
  "packageRules": [
    {
      "description": "Wait 3 days before creating a npm update PR",
      "matchDatasources": ["npm"],
      "stabilityDays": 3
    },
    {
      "description": "Automerge Prettier updates",
      "matchDepTypes": ["devDependencies"],
      "matchPackagePatterns": ["prettier"],
      "automerge": true
    },
    {
      "description": "Automerge minor/patch React monorepo updates",
      "extends": "monorepo:react",
      "groupName": "react monorepo",
      "matchUpdateTypes": ["minor", "patch"],
      "automerge": true
    }
  ]
```

Reply:
I'll think some more on this one, as I'm unsure what the descriptions would be that aren't repeating what's written in the `groupName` string.